homeassistant/components/roborock/switch.py

```diff
@@ -45,12 +45,23 @@ class RoborockSwitchDescription(
             RoborockCommand.SET_CHILD_LOCK_STATUS, {"lock_status": 1 if value else 0}
         ),
         get_value=lambda data: data.send(RoborockCommand.GET_CHILD_LOCK_STATUS),
-        evaluate_value=lambda data: data["lock_status"] == 1,
+        evaluate_value=lambda data: isinstance(data, dict) and data["lock_status"] == 1,
```

Comment:
I'd prefer not to have to do an isinstance check, but mypy complains otherwise. I could be missing an easy solution though

Reply:
Or potentially I should just create different Descriptions for a dict result of get_value vs an int result.